templates/admin/reporting/nav-filters.php

```diff
@@ -67,7 +67,7 @@
 					?>
 					<?php foreach ( $current_students as $id ) : ?>
 						<?php $s = get_user_by( 'id', $id ); ?>
-						<option value="<?php echo $id; ?>" selected="selected"><?php echo $s->display_name; ?> &lt;<?php echo $s->user_email; ?>&gt;</option>
+						<option value="<?php echo esc_attr( $id ); ?>" selected="selected"><?php echo $s->display_name; ?> &lt;<?php echo $s->user_email; ?>&gt;</option>
```

Comment:
I would escape display name and email too.

Reply:
done.